server/runtime/src/test/resources/configuration/ServerConfigurationParserTest.xml

```diff
@@ -73,21 +76,22 @@
                                         introspection-url="http://${org.infinispan.test.host.address}:14567/auth/realms/infinispan/protocol/openid-connect/token/introspect"
                                         client-id="infinispan-server"
                                         client-secret="1fdca4ec-c416-47e0-867a-3d471af7050f"/>
-
                </token-realm>
-               <truststore-realm path="truststore.p12" provider="SunJSSE"
-                                 relative-to="infinispan.server.config.path"
-                                 keystore-password="secret"/>
+               <truststore-realm/>
             </security-realm>
 
             <security-realm name="using-credentials">
                <server-identities>
                   <ssl>
                      <keystore path="ServerConfigurationParserTest-keystore.pfx"
                                relative-to="infinispan.server.config.path"
-                               alias="server">
+                               alias="server" provider="SunJSSE">
                         <credential-reference store="credentials" alias="keystore"/>
                      </keystore>
+                     <truststore path="truststore.p12" provider="SunJSSE"
+                                 relative-to="infinispan.server.config.path">
+                        <credential-reference store="credentials" alias="trust"/>
```

Comment:
<credential-reference> usage doesn't appear to be documented anywhere, neither for this use-case or it's use in general.

Reply:
https://github.com/infinispan/infinispan/pull/9070/files#diff-6e880072938cdbf4757fce0f304a80a85daf06c40eedcf2b8c83aa3563869b5fR15

Reply:
I meant the of credential-reference. This isn't an issue just with this PR, more of an omission from the docs in general.

Reply:
@ryanemerson We do, in fact, document credential-reference. Please see here: https://infinispan.org/docs/stable/titles/server/server.html#server_credentials

Reply:
Sorry I obviously missed that. Intellij didn't find anything when I hunted for `<credential-reference`.